packages/block-editor/src/components/block-popover/inbetween.js

```diff
@@ -144,16 +158,57 @@ function BlockPopoverInbetween( {
 		return {
 			top: previousRect ? previousRect.top : nextRect.top,
 			left: previousRect ? previousRect.right : nextRect.left,
-			right: nextRect ? nextRect.left : previousRect.right,
-			bottom: previousRect ? previousRect.bottom : nextRect.bottom,
+			right: previousRect ? previousRect.right : nextRect.left,
+			bottom: previousRect ? previousRect.left : nextRect.right,
```

Comment:
Just double-checking — Is this line correct? By following the same logic of the rest of the return values, it should have been `previousRect ? previousRect.top : nextRect.top`

Reply:
(asking also because of https://github.com/WordPress/gutenberg/pull/43691/files#r964934557)

Reply:
Hm.. it seems so, yes.. I've tested with your suggested values though and couldn't see any difference. 🤔 .

I also tried to test in your PR but it doesn't contain this PR. Can you rebase to test there as well? Since you're changing the logic in your PR, do you think it's okay to handle this there? I could create a follow up of course for this..

Reply:
> and couldn't see any difference. 🤔 .

That's expected, because passing `height: 0` kind of overrides the `bottom` value. 

In #43691, I decided to pass an actual `DOMRect` instead of a manually-created object — which should also remove any confusion. I would therefore ask you to take a closer look to the component while reviewing that PR 🙏